SPECS/libva/libva.spec

```diff
@@ -132,71 +133,48 @@ The library loads a hardware dependendent driver.
 This is the VA/X11 runtime library.
 
 %prep
-%setup -q -n %{_name}-%{version}
-# Add "libva-wayland%%{sover}" to baselibs.conf when enabling wayland build;
-# ugly I know ...This is needed since otherwise source validator
-#
-#   osc service run source_validator
-#
-# fails on sle
-echo libva-wayland%{sover} >> $RPM_SOURCE_DIR/baselibs.conf
+%autosetup -n %{_name}-%{version} -p1
 
 %build
-[ -d m4 ]  || mkdir m4
-autoreconf -v --install
-%configure \
+%meson \
+	-D driverdir=%{_libdir}/dri \
 %if %{build_gl}
-           --enable-glx \
-           --enable-wayland \
+	-D with_glx=yes \
+	-D with_x11=yes \
+	-D disable_drm=true \
+	-D with_wayland=no \
```

Comment:
I think we want with_wayland=yes to match the old version.

Reply:
The implementation of opensuse moved libva-wayland out of the %{build_gl} condition, libva-wayland always gets built so I think it is why `-D with_wayland=no \` is configured within gl mode here.
https://build.opensuse.org/request/show/1079220